presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1164,4 +1170,10 @@ public static boolean isListBuiltInFunctionsOnly(Session session)
     {
         return session.getSystemProperty(LIST_BUILT_IN_FUNCTIONS_ONLY, Boolean.class);
     }
+
+    @Deprecated
+    public static boolean isLegacyTaskPriorityTracking(Session session)
```

Comment:
Wait how can this be a session property? -- what about some query claims "TASK_FAIR" scheduling while some other query claim "QUERY_FAIR"? It looks like more like a cluster-level global config (e.g. configuration property)